packages/video_player_avplay/lib/video_player_platform_interface.dart

```diff
@@ -353,6 +363,48 @@ enum BufferConfigType {
   bufferingTimeoutInSecForPlay,
 }
 
+/// The different types of display rotations that can be set on the player.
+enum DisplayRotation {
+  /// No rotation.
+  rotation_0,
+
+  /// 90 degrees rotation.
+  rotation_90,
+
+  /// 180 degrees rotation.
+  rotation_180,
+
+  /// 270 degrees rotation.
+  rotation_270,
+}
+
+/// Sets the video screen mode in the specified display area.
+enum DisplayMode {
+  /// player display mode letter box
+  display_mode_letter_box,
+
+  /// player display mode origin size
+  display_mode_origin_size,
+
+  /// player display mode full screen
+  display_mode_full_screen,
+
+  /// player display mode cropped full
+  display_mode_cropped_full,
+
+  /// player display mode origin or letter
+  display_mode_origin_or_letter,
+
+  /// player display mode dst roi
+  display_mode_dst_roi,
+
+  /// player display mode auto aspect ratio
+  display_mode_auto_aspect_ratio,
+
+  /// player display mode dst roi auto aspect ratio
+  dispaly_mode_dst_roi_auto_aspect_ratio,
+}
```

Comment:
All values ​​in an enum must be written in camelCase.
And I think display_mode_ can be deleted because it is nested.